### DISCUSSION TITLE
How to correctly write IBM/DOS Image Files 3.5" HD

### DISCUSSION THREAD

**Discussion by NintenMSDOS** [keirf/greaseweazle#319] [Q&A]
That's about it. I'm having troubles trying to use Greaseweazle (and FluxMyFluffyFloppy) Because I'm not sure how to set up the profile. Following some tutorials I found the way to write Amiga (ADF) floppies using the profile of: Cylinders 0-80; Heads 0,1; Steps 1; and floppies works well with actual Amigas machine.

However, when I try to write IBM disk images, The program does write the disk correctly, but the disk are not readable by DOS compatible computers, and nowhere I found a tutorial with how to set-up a profile, since mostly are about amiga or other systems, not IBM/PC

Thank you in advance.

**Comment by keirf** [comment]
You don't really need a fancy profile. What sort of disk image are you writing?

**Reply by NintenMSDOS** [reply]
Games disk images, that are 1.440KB. 

`You don't really need a fancy profile.
`
So you mean I just set the "3.5-inch 1.44mb HighDensity DoubleSided (80 tracks)" template and leave the rest empty?

**Reply by keirf** [reply]
I think so. From the command line you'd simply do "gw write my1440.img". Problems could be if your media is not HD or your drive is not HD or if your media is bad.

**Reply by keirf** [reply]
If your image seems to write okay but is not recognised in a PC, are you confident the image is actually good?

**Reply by NintenMSDOS** [reply]
Yes, the image is actually good, since I've tried in virtual machine, and it does work well.

**Reply by keirf** [reply]
Are the drives known well aligned? Misalignment between them could cause issues.
